baitap_bien.ts

```diff
@@ -0,0 +1,74 @@
+console.log("Bai 1");
+//in dòng bên dưới
+console.log("Tôi năm nay 18 tuổi \nTôi có hoài bão để theo đuổi");
```

Comment:
```suggestion
const age = 18;
console.log("Tôi năm nay ${age} tuổi \nTôi có hoài bão để theo đuổi");

```
Need make variable for 18